app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,3 @@
+class PostsController < ApplicationController
+  def index; end
+end
```

Comment:
- [ ] Kindly add posts_controller #show actions, `def show; end`  
- It's okay for the actions to be empty for now 👍🏿